src/governance-votedb.cpp

```diff
@@ -89,6 +90,24 @@ std::set<uint256> CGovernanceObjectVoteFile::RemoveInvalidProposalVotes(const CO
     return removedVotes;
 }
 
+void CGovernanceObjectVoteFile::RemoveOldVotes(const CGovernanceVote& vote)
+{
+    vote_l_it it = listVotes.begin();
```

Comment:
My IDE recommends using auto for iterators, not sure if this is something you care to follow.